README.md

```diff
@@ -236,11 +236,11 @@ Here we use the variable `$it` to refer to the value being piped to the external
 
 Nu has early support for configuring the shell. You can refer to the book for a list of [all supported variables](https://www.nushell.sh/book/en/configuration.html).
 
-To set one of these variables, you can use `config --set`. For example:
+To set one of these variables, you can use `config set`. For example:
 
 ```shell
-> config --set [edit_mode "vi"]
-> config --set [path $nu.path]
+> config set [edit_mode "vi"]
+> config set [path $nu.path]
```

Comment:
these should be `config set edit_mode "vi"` and `config set path $nu.path` respectively.

Try `help config set` and same for the others to make sure it's correct. (in this case `config set` no longer takes a table as an argument but rather positionals.

Thanks!

Reply:
@andrasio I personally like the tables better than just a string of parameters. I wish we'd unify our terms. It's hard to remember when to do `[some thing]` and when to do `some thing`

Reply:
@andrasio I seem to have built the wrong version, as I still have the "old" commands.

Reply:
@morrme Could you also update the "Configuring Nu as a login shell" section of the [configuration page of the Nu book](https://www.nushell.sh/book/en/configuration.html) as well?

Reply:
> @morrme Could you also update the "Configuring Nu as a login shell" section of the [configuration page of the Nu book](https://www.nushell.sh/book/en/configuration.html) as well?

@tauheed-elahee sure!  This one should be wrapped up by tomorrow so expect a PR there soon!